### DISCUSSION TITLE
~jason.eckert/myblog/icon-computer/

### DISCUSSION THREAD

**Discussion by giscus** [jasoneckert/jasoneckert.github.io#2] [Announcements]
# ~jason.eckert/myblog/icon-computer/

My personal website and blog

https://triosdevelopers.com/~jason.eckert/myblog/icon-computer/

**Comment by jasoneckert** [comment]
I've received 94 emails about the ICON (and LEXICON) since this blog post was published. Now that I've added a comment section, people can add their comments here instead (however, emails are always welcome too!)
